.github/workflows/pre-release.yaml

```diff
@@ -7,43 +7,54 @@ on:
         description: 'Release commit'
         required: true
 
+permissions:
+  contents: read
+
 jobs:
   pre-release:
     name: Pre-release
     runs-on: ubuntu-20.04
 
+    permissions:
+      contents: write
```

Comment:
Do we need both of these?

Reply:
I'd say yes... but just because it would match the proposed standards, but I don't think it could have effective difference in this file tbh.

Reply:
I think what I'm confused about is that there is a single job in this workflow. Reading the docs: `You can use permissions either as a top-level key, to apply to all jobs in the workflow, or within specific jobs`, so we're setting it to read at the top-level, but then we change that same permission at the job level to something less restrictive.

Reply:
There's only one job in this workflow. Set it to `read` at the workflow level and remove this one.